syntax = "proto3";

package fraud_detection;

service FraudDetectionService {
  rpc CheckFraud (FraudCheckRequest) returns (FraudCheckResponse);
}

message FraudCheckRequest {
  string order_json = 1;
  string vector_clock_json = 2;
}

message FraudCheckResponse {
  string response_json = 1;
  string vector_clock_json = 2;
}
